HTTP_API_Guide.md

```diff
@@ -0,0 +1,196 @@
+HTTP API
+========
+
+The [Bloomberg Open API] provides access to market data. The
+[Bloomberg HTTP API] makes the [Open API] available via HTTP: clients POST
+commands to an HTTP server, which will respond with the relevant data.
+
+The [HTTP API] is currently in development, and does not yet provide access
+to all features of the [Open API]. At this time it supports the Reference Data
+Service (i.e. /blp/refdata) and the API Field Service (/blp/apiflds).
+
+This document provides an overview of the [HTTP API]: the URLs to use and the
+formatting for the HTTP requests and responses. We assume, for the purposes of
+this doc, that the server at http://blpapihost.example.com/ is running an
+implementation of this API.
+
+This doc also includes basic info about some of the underlying [Open API]
+operations. For more details on the [Open API], refer to the
+[BLPAPI Developer's Guide]. This document will often refer to specific sections
+of the [Developer's Guide] where appropriate.
+
+[Bloomberg Open API]: http://bloomberglabs.com/api
+[Open API]: http://bloomberglabs.com/api
+[Bloomberg HTTP API]: http://github.com/bloomberg/blpapi-http
+[HTTP API]: http://github.com/bloomberg/blpapi-http
+[BLPAPI Developer's Guide]: http://www.bloomberglabs.com/files/2014/07/blpapi-developers-guide-2.54.pdf
+[Developer's Guide]: http://www.bloomberglabs.com/files/2014/07/blpapi-developers-guide-2.54.pdf
+
+
+Requesting data
+---------------
+
+To request data, a client makes an HTTP POST request on a URL of the following
+form:
+
+```
+http://blpapihost.example.com/v1.0/request/blp/refdata/<operation>
+```
+
+Let's examine each segment of the path:
+
+* /v1.0 -- The version of the HTTP API we're using.
```

Comment:
As part of the refactoring yesterday, I remove the **/v1.0** out of the url. Sorry for the late reminder.